### PR TITLE
Update documentation for jhipster deployment to AWS

### PR DESCRIPTION
**Pull request by GeraldScott**
- Improve grammar
- Expand some details
- Replace deprecated security policy

### CODE REVIEW

File: pages/aws.md [hunk 1]
```diff
@@ -15,47 +15,61 @@ sitemap:
...
+Log into your Amazon AWS account and create an IAM user with programmatic access. 
 
-After that create a credentials file at `~/.aws/credentials` on Mac/Linux or `C:\Users\USERNAME\.aws\credentials` on Windows.
+Attach the following policies to grant the user the necessary permissions:
+- `AdministratorAccess-AWSElasticBeanstalk` (formerly `AWSElasticBeanstalkFullAccess`, now deprecated)
+- `AmazonRDSFullAccess` and
+- `IAMFullAccess`.
 
+Create the user and download the `csv` file with the new credentials. Use them to create a credentials file called `~/.aws/credentials` on Mac/Linux or `C:\Users\USERNAME\.aws\credentials` on Windows, as follows:
 ```
```
Comment (gmarziou) [inline pages/aws.md:40]: We could have syntax highlighting using `~~~ini`

```ini
[default]
aws_access_key_id = your_access_key
aws_secret_access_key = your_secret_key
```

**Review by gmarziou — COMMENTED** [review]
Great work, could you squash your 2 commits into 1 ?

**Comment by GeraldScott** [comment]
Resubmitted as one commit

**Comment by gmarziou** [comment]
Thanks but you did not have to open a new PR, you could just have squashed and force pushed to same branch, this PR would have been updated.

**Comment by GeraldScott** [comment]
I deleted my fork of the repo, so I had to redo it :roll_eyes:

**Comment by gmarziou** [comment]
Oh sorry for the extra work

**Pull request closed without merging**